samples/App.jsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Incorrect license header. License header should include the **year**. This issue is present in the all license headers in this PR. Please correct them all. Use [this](https://github.com/wso2/carbon-dashboards/blob/v4.0.0.alpha4/samples/widgets/CpuUsage/webpack.config.js#L1-L17) license header.